### ISSUE TITLE
Page for Database Provider "PlanetScale"

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2-docs#2671]
to replace https://github.com/prisma/prisma/issues/7292

Additionally to the measures defined in https://github.com/prisma/docs/issues/2448

**Comment by tanberry** [comment]
Thank you @Jan, for the draft content for the "Database Provider: PlanetScale" in [Notion](https://www.notion.so/prismaio/Database-Provider-PlanetScale-261f989bf62745dd8446e0485f180bf1) !

Is the thought to have this as a new type of doc, under Guides? Or what is your thought on where this material fits? To me, it seems perfect content for an entire guide on developing with PlanetScale...

**Comment by janpio** [comment]
We do not have similar content yet, and it is definitely "guide"y so the Guides section sounds like a good fit. 

We talked about this in the past as "Using Prisma with x" or "Database Providers" (I actually started an issue for these to collect potential database services to document: https://github.com/prisma/docs/issues/2667).

Right now the priority should only be to get the PlanetScale information into the docs - if finding the right place takes more time, we can just put it under "Guides" with the title "Prisma + PlanetScale" and move it later.
